Symfony/CS/AbstractPhpdocTypesFixer.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS;
+
+use Symfony\CS\DocBlock\DocBlock;
+use Symfony\CS\DocBlock\Line;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Graham Campbell <graham@mineuk.com>
+ */
+abstract class AbstractPhpdocTypesFixer extends AbstractFixer
+{
+    /**
+     * The annotation tags search inside.
+     *
+     * @var string[]
+     */
+    private static $tags = array('param', 'return', 'type', 'var', 'property');
```

Comment:
so you want me to make everything protected in an abstract class rather than private? that seems a little strange, since an abstract class can still define internal methods that it wants to keep private?